### PR TITLE
Use milestone=fcs only on JDK8 release

### PR DESCRIPTION
**Pull request by M-Davies**
Closes: #2188 
Signed-off-by: Morgan Davies <morgandavies2020@gmail.com>

### CODE REVIEW

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

File: sbin/build.sh [hunk 1]
```diff
@@ -174,7 +174,7 @@ configureVersionStringParameter() {
   echo "OpenJDK repo tag is ${openJdkVersion}"
 
   # --with-milestone=fcs deprecated at jdk11, removed at jdk12
-  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -lt 12 ]; then
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -lt 12 ] && [ "${BUILD_CONFIG[RELEASE]}" == "true" ]; then
```
Comment (andrew-m-leonard) [inline sbin/build.sh:177]: Although strictly speaking jdk11 still supports it, we use without-version-pre/opt

File: sbin/build.sh [hunk 2]
```diff
@@ -173,8 +173,8 @@ configureVersionStringParameter() {
   local openJdkVersion=$(getOpenJdkVersion)
   echo "OpenJDK repo tag is ${openJdkVersion}"
 
-  # --with-milestone=fcs deprecated at jdk11, removed at jdk12
-  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -lt 12 ]; then
+  # --with-milestone=fcs deprecated at jdk12+ and not used for jdk11- (we use --without-version-pre/opt)
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" == 8 ] && [ "${BUILD_CONFIG[RELEASE]}" == "true" ]; then
     addConfigureArg "--with-milestone=" "fcs"
```
Comment (karianna) [inline sbin/build.sh:178]: This logic will fail for 9 and 10 no?

**Review by karianna — CHANGES_REQUESTED** [review]

**Comment by M-Davies** [comment]
run tests

File: sbin/build.sh (hunk 2 above)
Comment (M-Davies) [reply sbin/build.sh:178]: That's the idea as we [shouldn't be doing this on nightly builds for any version](https://github.com/AdoptOpenJDK/openjdk-build/issues/2188#issuecomment-718971081), only releases. If the user wants this to be in the pipeline for whatever reason, they can specify it in via the Jenkins params or `./makejdk-any-platform.sh -C --with-milestone=fcs`

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**